src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle color: " + getColor()
+                + " area: " + getArea() + " sq.units "
+                + " radius: " + getRadius());

```

Comment:
```suggestion
                + " radius: " + radius);
```
if you have an opportunity to use field directly, you don't need to call its get method, fix it in every figure implementation